mysql-test/main/func_str.test

```diff
@@ -2193,3 +2193,51 @@ SELECT GROUP_CONCAT( UpdateXML( '<a>new year</a>', '/a', '2019-01-01 00:00:00' )
 --echo #
 --echo # End of 10.4 tests
 --echo #
+
+--echo #
+--echo # MDEV-25704 Function random_bytes
+--echo #
+
+--source include/have_sequence.inc
+
+create table t1 as select random_bytes(100);
```

Comment:
I'm sorry, should've said it in the first comment. Put `show create table t1` here. The point is not just to do `create table` but to see what column type is created for `random_bytes(100)`